src/main/java/org/myrobotlab/service/InMoov2Arm.java

```diff
@@ -135,10 +135,15 @@ public InMoov2Arm(String n, String id) throws Exception {
     shoulder.setPin(10);
     omoplate.setPin(11);
 
-    bicep.setMinMax(5.0, 90.0);
-    rotate.setMinMax(40.0, 180.0);
+    bicep.setMinMax(0.0, 180.0);

```

Comment:
These shouldn't have changed..  The bicep min/max should be 5-90 ... not 0 to180.  The change in behavior in setMinMax was fixed .. these changes should be reverted.

Reply:
setMinMax is deprecated on last_servo_fixes branch.  It's behavior is to set both the input and the output min/max values for the servos.  setMinMax(5.0,90.0)  is the same as, and should be replaced with the more explicit map(5,90,5,90)   .

there's been an issue with autoDisable for a while.  I'll try to work through getting that fixed on last_servo_fixes branch.. at that point, hopefully all of this stuff is straightened out. 

There were many years of bad variable names and unnecessary logic that have piled up in the Servo class.  It's been a significant effort to unroll it all and clean it up.  I think GroG & I are very close though.

Reply:
Hello Kevin,
The way it should be is :
 bicep.setMap(5.0, 90.0, 0.0, 180.0);
Because we need to set the input values as default, the output values need to be done by the user.
I was trying to fix that in the InMoov2Arm.java but unfortunately setMap was returning a error when compiling.
So because it was not compiling correctly I just set back as it was and added annotations.
When you say setMinMax is fixed and then you say it is deprecated, does it mean that setMinMax represent now the input or the output, or still both together?